docker-start-chain.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+password=admin123
+
+(echo "y"
+sleep 1
+echo $password
+sleep 1
+echo $password)|aelf-command create
```

Comment:
Use a fixed private key for testing. No need to generate every time.

Reply:
If we want to use a fixed private key. There are probably two ways we can do this. One is developers to install the aelf-command, and create account locally. Another is we put a fixed private key to our docker image.